api.go

```diff
@@ -832,10 +830,11 @@ func (r *Raft) AddPeer(peer ServerAddress) Future {
 	}, 0)
 }
 
-// RemovePeer (deprecated) is used to remove a peer from the cluster. If the
-// current leader is being removed, it will cause a new election
-// to occur. This must be run on the leader or it will fail.
-// Use RemoveServer instead.
+// RemovePeer from the cluster configuration. If the current leader is being
```

Comment:
Missing a word in first sentence?

Reply:
I couldn't find many examples of this style, but I think they exist somewhere. Maybe I'll stumble on them again sometime. This style uses the method name as part of the sentence, so "remove peer from the cluster configuration".